PowerPointLabs/PowerPointLabs/FrameMotionAnimation.cs

```diff
@@ -92,10 +103,10 @@ public static void AddZoomToAreaPanFrameMotionAnimation(PowerPointSlide animatio
             float incrementLeft = (finalX - initialX) / numFrames;
             float incrementTop = (finalY - initialY) / numFrames;
 
-            AddFrameAnimationEffects(animationSlide, initialShape, incrementLeft, incrementTop, incrementWidth, incrementHeight, 0.0f, 0.0f, duration, numFrames);
+            AddFrameAnimationEffects(animationSlide, initialShape, incrementLeft, incrementTop, incrementWidth, incrementHeight, false, false, 0.0f, 0.0f, duration, numFrames);

```

Comment:
If I add the prefix I must also add them for the other 4 params, is that okay?

![image](https://cloud.githubusercontent.com/assets/19278089/22320612/b78b1e0a-e3c8-11e6-8f1d-4bf7ace9d927.png)